bin/validate_schema.mjs

```diff
@@ -32,24 +32,34 @@ const argv = yargs(hideBin(process.argv))
 
 // 1) Validate common params and suffixes
 const mainDir = argv.dirPath;
+const pixelsDir = path.join(mainDir, 'pixels');
 const commonParams = JSON5.parse(fs.readFileSync(`${mainDir}/common_params.json`));
 const commonSuffixes = JSON5.parse(fs.readFileSync(`${mainDir}/common_suffixes.json`));
 const validator = new DefinitionsValidator(commonParams, commonSuffixes);
 logErrors('ERROR in common_params.json:', validator.validateCommonParamsDefinition());
 logErrors('ERROR in common_suffixes.json:', validator.validateCommonSuffixesDefinition());
 
 // 2) Validate pixels and params
-const pixelsDir = path.join(mainDir, 'pixels');
 function validateFile(file) {
     console.log(`Validating pixels definition: ${file}`);
-    const pixelsDef = JSON5.parse(fs.readFileSync(path.join(pixelsDir, file)));
+    const pixelsDef = JSON5.parse(fs.readFileSync(file));
     logErrors(`ERROR in ${file}:`, validator.validatePixelsDefinition(pixelsDef));
 }
 
+function validateFolder(folder) {
+    fs.readdirSync(folder).forEach((file) => {
```

Comment:
Since node 20, `readdir` has a `recursive` option, which we could use here: https://nodejs.org/api/fs.html#fsreaddirsyncpath-options